src/test/java/io/reactivex/observers/SerializedObserverTest.java

```diff
@@ -260,7 +260,6 @@ public void runConcurrencyTest() {
      *
      * @throws InterruptedException if the await is interrupted
      */
-    @Ignore("this is non-deterministic ... haven't figured out what's wrong with the test yet (benjchristensen: July 2014)")
```

Comment:
This breaks on the CI, remove it completely or restore ignore.

Reply:
I restored the `ignore`.